### DISCUSSION TITLE
Artillery installation fails with downloading chromium error

### DISCUSSION THREAD

**Discussion by anildel8s** [artilleryio/artillery#2422] [Artillery]
When i run the usual
npm i artillery
getting the below error
npm ERR! code 1
npm ERR! path C:\Users\USER\AppData\Roaming\npm\node_modules\artillery\node_modules\@playwright\browser-chromium
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c node install.js
npm ERR! Downloading Chromium 119.0.6045.9 (playwright build v1084) from https://playwright.azureedge.net/builds/chromium/1084/chromium-win64.zip
npm ERR! Downloading Chromium 119.0.6045.9 (playwright build v1084) from https://playwright-akamai.azureedge.net/builds/chromium/1084/chromium-win64.zip
npm ERR! Downloading Chromium 119.0.6045.9 (playwright build v1084) from https://playwright-verizon.azureedge.net/builds/chromium/1084/chromium-win64.zip
npm ERR! Error: Download failed: server returned code 403 body '<html>

Tried installing globally as well but same issue persists.  I am not sure why playwright is downloading and I think it should be part of the dependency artillery-engine-playwright.  The thing is I dont need the playwright part, just the APIs that I need to test.  But there is no way to skip this.  Please advice or point me to any material around this.

**Comment by bernardobridge** [comment]
Hello 👋 ,

This seems to be an issue with pulling the browser binary from Playwright's build archive. I just tried the download link myself (https://playwright.azureedge.net/builds/chromium/1084/chromium-win64.zip), and it seems to work for me.

- Can you please tell me what OS you're on (e.g. is it `win32`, `win64`, etc?);
- Have you tried again since the issue started? It could have been an intermittent issue on their side;
- However, it seems they have 2 backup CDNs (akamai and verizon), so I find it unlikely it was an intermittent issue. Do you perhaps have any sort of network firewall or similar that could be preventing the download?

**Reply by anildel8s** [reply]
Hi, Thanks for the response.  Its a win64 machine and retried lot of times and even tried installing playwright browsers using npx first but still its failing.  Did a bit of digging in and it looks to be the 3rd point and we are investigating the network firewall option.
